src/Command/CreateControllerCommand.php

```diff
@@ -6,25 +6,16 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use GuzzleHttp\ClientInterface;
 
 class CreateControllerCommand extends Command {
 
-    private $client;
+    const CONTROLLER_DIRECTORY = 'src/Controller/';
```

Comment:
@vladimino  I FIgured running `php bin/console ` from root but `php console ` from bin folder is not working with me yet as 
`const CONTROLLER_DIRECTORY = __DIR__ . '/../src/Controller/'` is not working so for now only first option works .

Reply:
Last commit solves this.